src/config/ecencyApi.ts

```diff
@@ -2,13 +2,15 @@ import axios from 'axios';
 import Config from 'react-native-config';
 import VersionNumber from 'react-native-version-number';
 import { get } from 'lodash';
+import Reactotron from 'reactotron-react-native';
```

Comment:
We do no use `Reactotron` imports directly in production code, as production build strips reactotron-react-native package and cause app to crash if not in debug mode... there is a workaround to it by using `console.tron` but I have not configured it in this project yet. we usually use regular` console.log`, `console.debug` for now as it is configured to be removed before production.